zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -61,6 +61,11 @@ final case class Response private (
    */
   def withServerTime: Response = self.copy(attribute = self.attribute.withServerTime)
 
+  /**
+   * Wraps the current response as a Http
+   */
+  def wrapHttp: Http[Any, Nothing, Any, Response] = Http.succeed(self)
```

Comment:
Unit test missing

Reply:
Could we call it `toHttp`, like `ZIO#toLayer`, `ZIO#toManaged`, etc.?